pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -47,6 +47,12 @@ const (
 	ClosedIndexField = "closed"
 )
 
+// Registry deployment constants
+const (
+	MigRegistryDefaultLivenessTimeout  = 3
+	MigRegistryDefaultReadinessTimeout = 3
```

Comment:
The above two values don't seem to be used anywhere. It looks like `RegistryDefaultProbeTimeout` is used instead. Should these be removed?

Reply:
@sseago yeah forgot to remove them. will do now